README.md

```diff
@@ -1,6 +1,6 @@
 --- 
 title: Flow GO SDK 
-description: Packages for Go developers to build applications that interact with the Flow network.
+description: Packages for Go developers to build applications that interact with the Flow network
 ---
```

Comment:
Does the README need to have frontmatter? I don't think this page is ingested by Gatsby, is it? 

I'd prefer to remove the frontmatter if possible because it doesn't look so good when viewed directly on GH